include/wx/gtk/textctrl.h

```diff
@@ -10,6 +10,11 @@
 #ifndef _WX_GTK_TEXTCTRL_H_
 #define _WX_GTK_TEXTCTRL_H_
 
+#include <gtk/gtk.h>
+#if GTK_CHECK_VERSION(3, 24, 0)
+#include <gtk/gtkunixprint.h>
```

Comment:
This header seems wrong, why should we need printing stuff here?

Reply:
You're right.
Only `#include <gtk/gtk.h>` is required.
https://developer.gnome.org/gtk3/stable/GtkEventControllerKey.html
https://developer.gnome.org/gtk3/3.24/GtkEventControllerKey.html

My devhelp manual is wrong.
![Screenshot from 2019-06-25 10-12-19](https://user-images.githubusercontent.com/43208265/60061877-c83ad000-9731-11e9-9c15-67a62a73f634.png)